config/orte_check_lsf.m4

```diff
@@ -137,7 +201,7 @@ AC_DEFUN([ORTE_CHECK_LSF],[
            $1_WRAPPER_EXTRA_LDFLAGS=[$]$1_LDFLAGS
            $1_WRAPPER_EXTRA_LIBS=[$]$1_LIBS
            $2],
-          [AS_IF([test ! -z "$with_lsf" && test "$with_lsf" != "no"],
+          [AS_IF([test ! -z "$with_lsf" && test "$with_lsf" != "no" && test "$orte_check_lsf_event_conflict" = "yes"],
```

Comment:
(Note from code review with Geoff)
This check is not doing what I originally thought and should be removed.

The scenario that I was considering is if LSF and Libevent are the default search path (no specified with --with-lsf) then if we detect a conflict then we error out after printing the warning. When actually it is probably ok to just print the warning and not error out here since the user did not explicitly say `--with-lsf`. They can notice it was not compiled in the final list of runtime components reported, then scroll up to see the warning message indicating the possible issue.

Reply:
done